userguide/tasks.rst

```diff
@@ -1676,18 +1676,36 @@ shows the configuration options for Cloud Syncs.
+   | Remote Encryption   | checkbox       | Use `rclone Crypt <https://rclone.org/crypt/>`__ to manage data encryption during *PUSH* or *PULL*         |
+   |                     |                | transfers:                                                                                                 |
+   |                     |                |                                                                                                            |
+   |                     |                | *PUSH:* Encrypt files before transfer and store the encrypted files on the remote system. Files are        |
+   |                     |                | encrypted using the :guilabel:`Encryption Password` and :guilabel:`Encryption Salt` values.                |
+   |                     |                |                                                                                                            |
+   |                     |                | *PULL:* Decrypt files that are being stored on the remote system before the transfer. Transferring the     |
+   |                     |                | encrypted files requires entering the same :guilabel:`Encryption Password` and :guilabel:`Encryption Salt` |
+   |                     |                | that was used to encrypt the files.                                                                        |
+   |                     |                |                                                                                                            |
+   |                     |                | Adds the :guilabel:`Filename Encryption`, :guilabel:`Encryption Password`, and :guilabel:`Encryption Salt` |
+   |                     |                | options.                                                                                                   |
    +---------------------+----------------+------------------------------------------------------------------------------------------------------------+
-   | Filename Encryption | checkbox       | Encrypt the shared file names. Only appears when :guilabel:`Remote encryption` is enabled.                 |
+   | Filename Encryption | checkbox       | Include the filename with the :guilabel:`Remote Encryption` operation:                                     |
+   |                     |                |                                                                                                            |
+   |                     |                | *PUSH:* Encrypt the file name in addition to the file contents.                                            |
+   |                     |                |                                                                                                            |
+   |                     |                | *PULL:* Decrypt the file name when the file is transferred from the remote system. Decrypting the file     |
```

Comment:
The cause of this very ticket was to say here:

"When pulling with filename decryption and specifying incorrect password or salt, no files will be pulled but operation will be considered successful. "

Reply:
On this point - Is this style of message (success, but nothing happened) immutable or is there some idea that this will be changed?

Reply:
It will look like the remote was empty and there is nothing we can do about it.